macros/SpecName.ejs

```diff
@@ -848,24 +836,24 @@ var specList = {
         url  : 'http://dev.w3.org/2006/webapi/selectors-api2/'
     },
     'Shadow DOM':{
-        name : 'Shadow DOM',
-        url  : 'https://www.w3.org/TR/shadow-dom/'
+        name : 'DOM (Shadow trees)',
+        url  : 'https://dom.spec.whatwg.org/#shadow-trees'
     },
     'Shape Detection API': {
         name : 'Accelerated Shape Detection in Images',
         url  : 'https://wicg.github.io/shape-detection-api/'
     },
     'Shared Memory':{
-        name : 'ECMAScript Shared Memory and Atomics',
-        url  : 'http://tc39.github.io/ecmascript_sharedmem/shmem.html'
+        name : 'ECMAScript (Memory Model)',
```

Comment:
I replaced calls to 'Shared Memory' with calls to 'ESDraft'. We can remove the 'Shared Memory' entry entirely.